vscode/syntaxes/qsharp.tmLanguage.json

```diff
@@ -42,11 +39,11 @@
       "patterns": [
         {
           "name": "keyword.control.qsharp",
-          "match": "\\b(use|using|borrow|borrowing|mutable|let|set|if|elif|else|repeat|until|fixup|for|in|while|return|fail|within|apply)\\b"
+          "match": "\\b(use|borrow|mutable|let|set|if|elif|else|repeat|until|fixup|for|in|while|return|fail|within|apply)\\b"
         },
         {
           "name": "keyword.other.qsharp",
-          "match": "\\b(new|not|and|or|w/)\\b"
+          "match": "\\b(not|and|or)\\b"
```

Comment:
`\b` is a word boundary, which is why the regex will never match the "boundary" between the `/` and the following whitespace. Try:

```suggestion
          "match": "\\b(not|and|or)\\b|\\b(w/)"
```

Note this also matches the `w/` in something like `w/i`, but as far as I can tell, that is valid Q# (parsed as two separate tokens, `w/` and `i`). If we ever wanted to disallow adjacent identifiers like that, we could put a whitespace character in the regex.

Reply:
Ooo, that does work! But oddly, doesn't seem to work for `w/=` where only the `w/` gets highlighted despite adding `|\\b(w/=)` to the string. That said, `=` is never highlighted when it's part of a combined operation and assignment, so in `or=` only the `or` is highlighted. So maybe just highlighting the `w/` is sufficient and consistent?

Reply:
Ok, with the latest update it will highlight both the `w/` and the `<-` as keywords:
<img width="225" alt="image" src="https://github.com/microsoft/qsharp/assets/10567287/814ed4f8-14c1-44de-b768-b5370d75ece3">

Reply:
Wait, but if we're marking `<-` as a keyword, why not all the other operators? Why not, say, `->` or even `==` at that point?

Also, now that I'm checking the docs, `keyword.operator` seems to be the more appropriate scope for these. https://github.com/dunstontc/textmate/blob/master/scopes.md#keyword

Reply:
Honestly, I think we should, especially with the further distinction as `keyword.operator`. I'll look through the parser and try to round up all the operators we support.

Reply:
Ok, I've added more operators and they seem to be doing the right thing. I still need to add a few more, specifically some of the triple symbol binary operators, and then I should have a full set.